test/components/toggle.spec.tsx

```diff
@@ -0,0 +1,69 @@
+import * as React from 'react';
+import {expect, ClientRenderer, sinon, simulate, waitFor} from 'test-drive-react';
+
+import {Toggle} from '../../src';
+
+describe('<Toggle/>', () => {
```

Comment:
I don't see any demo / component test. We're trying to have a tested demo for each component that shows a relevant scenario with side effects.